projects/distributed-tracing/src/shared/dashboard/widgets/table/table-widget-column.model.ts

```diff
@@ -65,6 +65,13 @@ export class TableWidgetColumnModel {
   })
   public filterable?: boolean = false;
 
+  @ModelProperty({
```

Comment:
I'm really not a fan of introducing this concept at the generic table column level. Can we switch to whole different set of columns for a flattened list? Perhaps create a new flattenable tree table data source or something?

Reply:
My suggestion would be something like (in pseudo-model):

```
{
   type: 'flattenable-tree-table',
   flat: {
       type: 'entity-table-data-source',
       ...
   }
   tree: {
       type: 'entity-table-data-source',
       ...
   }
}

```